src/makers/kerlink/hotspots.ts

```diff
@@ -0,0 +1,73 @@
+import HotspotIcon_iFemto from '@ifemto.svg'
+import HotspotIcon_iFemtoEvo from '@ifemto_evo.svg'
+import HotspotIcon_iStation from '@istation.svg'
+import { MakerHotspot } from '../hotspotMakerTypes'
+import ANTENNAS from './antennas'
+
+const qrKerlink = [
+  'Please visit the following URL from your computer or directly your phone by selecting the link below:',
+  'https://helium-onboarding.kerlink.com/?wallet=WALLET|helium-onboarding.kerlink.com',
+]
+
+const KERLINK_IFEMTO = {
+  name: 'Kerlink iFemtoCell',
+  onboardType: 'QR',
+  icon: HotspotIcon_iFemto,
+  translations: {
+    en: {
+      diagnostic:
+        '<b><white>Diagnostic support allows Kerlink to identify issues with your Hotspot in a secure way.</white></b>\n\nKerlink will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
+      power: [
+        'Attach the antenna and plug in the provided power adapter.',
+        'Your Hotspot will boot up, and its light will become Green when ready.',
+      ],
+      qr: qrKerlink,
```

Comment:
I don't like a url being tied to the translation and it makes it a bit obtuse to understand what the first and second position are for. I think it would be better to be something like...

```ts
const qr =
  'Please visit the following URL from your computer or directly your phone by selecting the link below:'
const qrLink =
  'https://helium-onboarding.kerlink.com/?wallet=WALLET|helium-onboarding.kerlink.com'

const KERLINK_IFEMTO = {
  name: 'Kerlink iFemtoCell',
  onboardType: 'QR',
  icon: HotspotIcon_iFemto,
  qrLink,
  translations: {
    en: {
      diagnostic:
        '<b><white>Diagnostic support allows Kerlink to identify issues with your Hotspot in a secure way.</white></b>\n\nKerlink will never have access to private keys and will only ever be able to access your Hotspot and not any other devices on your Network.\n\nIf you would like to opt-out of diagnostic support please email <purple><b>support@helium.com</b></purple> from the email used to purchase the Hotspot.',
      power: [
        'Attach the antenna and plug in the provided power adapter.',
        'Your Hotspot will boot up, and its light will become Green when ready.',
      ],
      qr,
    },
  },
  antenna: {
    default: ANTENNAS.KERLINK_IFEMTO,
  },
} as MakerHotspot
```

Reply:
I didn't like it either, but I'm so bad at javascript that this was the best I could do.